backend/packages/Upgrade/src/api/controllers/validators/ExperimentUserValidator.ts

```diff
@@ -0,0 +1,76 @@
+import { IsNotEmpty, IsOptional, IsString, ValidationOptions, isObject, registerDecorator } from 'class-validator';
+
+export const IsRecord = (validationOptions?: ValidationOptions) => {
```

Comment:
IsRecord is too generic name for the specific Group. Better name should be IsGroupRecord

Reply:
Code updated